content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Authentication Process to obtain the Access Token
+
+Data Services Cloud Console public API relies on OAuth 2.0 third party authorization framework on behalf of the resource owner (DSCC user) for security. User starts by authenticating into HPE GreenLake which is accommodated by the Identity Provider (validated by username, password, MFA or SSO). Using the API Gateway menu in HPE GreenLake customer registers their application to obtain the OAuth 2.0 API client credentials. This association allows user to obtain the Access Token.
+
+The access token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the application must use the refresh API token and the refresh API to obtain the valid access tokens
+
+![Access API process](/img/api-access-token-complete-path.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, user must login into the [GreenLake Common Cloud](https:\common.cloud.hpe.com), and deployed the Data Services Common Cloud Console Application into the organization that is associated with the user account. The **API** **Gateway** menu is available inside **Manage Account** menu.
```

Comment:
the Data Services Common Cloud Console Application. Is it different app from "Data Services Cloud Console" app? If not, you might want to use the "Data Services Cloud Console" application throughout the post for consistency.
If the Data Services Common Cloud Console app is different than the Data Services Cloud Console app, then you might want to give a brief description of what is the Data Services Common Cloud Console app vs the Data Services Cloud Console app.